PlayCover/Views/Sidebar Views/IPALibraryView.swift

```diff
@@ -20,98 +20,107 @@ struct IPALibraryView: View {
     @State private var addSourcePresented = false
 
     var body: some View {
-        ZStack {
-            ScrollView {
-                if !isList {
-                    LazyVGrid(columns: gridLayout, alignment: .center) {
-                        ForEach(storeVM.filteredApps, id: \.bundleID) { app in
-                            StoreAppView(selectedBackgroundColor: $selectedBackgroundColor,
-                                         selectedTextColor: $selectedTextColor,
-                                         selected: $selected,
-                                         app: app,
-                                         isList: isList)
-                            .environmentObject(DownloadVM.shared)
+        if #available(macOS 13.0, *) {
```

Comment:
if you are using if available, why you need to bump macs requirement? on other macos version you can just simply start download.
I'm just asking

Reply:
For some reason, SwiftUI requires the View to be available on macOS 12

Reply:
then if you bumped requirements, why did you use if available? 🥴